client/src/scenes/game.ts

```diff
@@ -168,10 +168,18 @@ export class GameScene extends Phaser.Scene {
     });
   }
 
-  private addPlayer({ id, x, y }: Player) {
+  private addPlayer({ id, x, y, name }: Player) {
     const sprite = new Phaser.GameObjects.Sprite(this, x, y, "player").setOrigin(0.5, 1);
+    const nameText = new Phaser.GameObjects.Text(this, x, y, name, {
```

Comment:
don't you want `y - sprite.height` or something?

Reply:
yea